### DISCUSSION TITLE
기존 스터디 맴버 3기 참여 안내

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#616] [공지사항]
스터디 1기와 2기에 참여하신 기존 맴버 중에서 지원자를 받아서 다음 3가지 형태로 다음 기수에 기여하실 수 있는 기회를 드리려고 합니다.

## 재수
- 다시 신입의 마음으로 리트코드 75문제를 한 번 더 풀고 싶은 분
- 풀었던 문제가 실제 코딩 테스트에 나왔을 때 스스로 해결하실 자신이 아직 없으신 분
- 과거 기수 활동을 돌아보았을 때, 매주 마감에 쫓겨 문제에 대해서 충분히 고찰하시지 못하신 분들께 추천

## 코치 (선배, 멘토)
- 동일한 15주의 과정을 먼저 겪어 본 선배의 입장에서 코드 리뷰 및 질의 응답에 적극적으로 참여하시고 싶은 분
- 다양한 답안 코드를 접하면서 문제에 접근하는 새로운 관점과 통찰을 얻고 싶은신 분
- 과거 기수 활동을 나름 충실하게 하셨고 스터디를 통해 얻으신 좋은 경험을 새로운 스터디 맴버 분들께 나누시고 싶은 분들께 추천

## 리더
- 스터디를 주도적으로 이끄고 싶으신 분
- 코드 저장소와 커뮤니티 관리를 하실 수 있도록 깃허브와 디스코드에서 maintainer 권한을 부여
- 달레 커뮤니티의 운영진으로서 그 밖에 각종 특해 ㅋㅋ

어떤 형태로든 3기 스터디를 도와주실 의향이 있으신 분들은 아래 댓글로 달아주시면 다음 기수 계획을 세우는데 도움이 될 것 같습니다! 🙏

**Comment by obzva** [comment]
코치로 참여하여 PR 등의 형태로 3기 분들께 도움 드리고 싶습니다 ㅎㅎ

**Comment by HC-kang** [comment]
코치로 참여해서 다른분들 풀이도 보고, 종종 다른 언어로 다시 풀어보고싶기도 한데 괜찮을까요?

**Reply by DaleSeo** [reply]
@HC-kang 님, 그럼요. 2기 때 @bky373 님도 비슷하게 활동하셨습니다.

**Comment by EgonD3V** [comment]
코치 참여 및 추가적인 알고리즘 문제(Grind 75 or NeetCode 150) 스터디 리더 지원드립니다.

**Comment by SamTheKorean** [comment]
리더로 참여하여 스터디 이끌며 저도 같이 배우고 싶습니다!

**Comment by TonyKim9401** [comment]
재수 & 코치 지원하고 싶습니다. 리더는 정기적으로 무언가 해야하는지 확인 후에 지원해보고 싶습니다 :)

**Comment by DaleSeo** [comment]
깃허브와 디스코드에서 @SamTheKorean과 @TonyKim9401 님께 리더 권한을 부여하고 @obzva, @HC-kang, @EgonD3V 님께 코치 권한을 부여드렸습니다. 3기 스터디 참여자분들이 좋은 경험을 하고 건강한 스터디 문화가 만들어질 수 있도록 좋은 활동을 부탁드리겠습니다. 🙇‍♂️

**Reply by obzva** [reply]
감사합니다 ㅎㅎ 화이팅..!

**Reply by TonyKim9401** [reply]
열심히 해보겠습니다. 부족한 부분은 편하게 피드백 부탁드립니다!

**Comment by yolophg** [comment]
안녕하세요, 저는 이번 3기에선 재수로 참여하여 리트코드 풀이활동에 집중해서 다시 한 번 밀도있게 문제들을 풀어보고 싶습니다!🔥

**Comment by jaejeong1** [comment]
[재수]
안녕하세요~! 2기에 이어 3기에 재수로 참여해 못풀었던 문제들을 다시 풀어보고 싶습니다!! 잘부탁드립니다 :)

**Comment by TonyKim9401** [comment]
3기 재참여 요청하신 분들 모두 인비테이션 발송해드렸습니다.
이메일 확인 부탁드립니다! 3기에서도 모두 잘 부탁드립니다!
